src/reloading.js

```diff
@@ -123,12 +114,20 @@ function loader(mappings, entryPoints, options) {
         throw error;
       }
 
-      var hook = scope.reloadHooks[id];
       var module = cache[id] = {
         exports: {},
-        __accepted: false,
-        onReload: function (hook) {
-          scope.reloadHooks[id] = hook;
+        hot: {
+          onUpdate: function (maybe, hook) {
+            var realHook = hook;
+            if (!realHook) {
+              realHook = maybe;
+            } else {
+              console.warn(`LiveReactload: You are providing two arguments to the module.hot.onUpdate hook, and we are
```

Comment:
Well, I think `console.error` indicates that something broke... since we're no longer using `accept` maybe we shouldn't accept two arguments at all though.